WalletWasabi.Gui/Helpers/AvalonStudioShellExtensions.cs

```diff
@@ -17,25 +17,9 @@ public static class AvalonStudioShellExtensions
 	{
 		// Replacement of avalonia's https://github.com/VitalElement/AvalonStudio.Shell/blob/eaa708e6c1428afd142e364dc6dbbd7ff5ba69dc/src/AvalonStudio.Shell.Extensibility/Shell/IShellExtensions.cs
 		// because we want to use IoC to create T
-		public static T GetOrCreate<T>(this IShell me) where T : IDocumentTabViewModel
```

Comment:
Please explain.

Reply:
I backported this code into AvalonStudio.Shell, as it maybe useful elsewhere.

Reply:
Is the comment above the method declaration still valid?

Reply:
If AvalonStudio has it without the `IDocumentTabViewModel` constraint then it makes no sense for us to have this constraint as it can be done anyway. I removed this function altogether.